### ISSUE TITLE
fusion_dataset.py

### ISSUE THREAD

**Issue by leeqiaogithub** [ericsujw/InstColorization#8]
hi.I am confused of the meaning of box_info_2x/ box_info_4x/ box_info_8x in fusion_dataset.py.Any help would be greatly appreciated

**Comment by iamxinxin** [comment]
可能是为了标记 在不同卷积层里每个instance对应于 full image的位置，让instance模型的参数和相应位置的整图特征能刚好对应上。

FusionGenerator.forward in fusion_model.py  
      `

        conv1_2 = self.model1(torch.cat((input_A,input_B,mask_B),dim=1))
        conv1_2 = self.weight_layer(instance_feature['conv1_2'], conv1_2, box_info_list[0])

        conv2_2 = self.model2(conv1_2[:,:,::2,::2])
        conv2_2 = self.weight_layer2(instance_feature['conv2_2'], conv2_2, box_info_list[1])

        conv3_3 = self.model3(conv2_2[:,:,::2,::2])
        conv3_3 = self.weight_layer3(instance_feature['conv3_3'], conv3_3, box_info_list[2])

        conv4_3 = self.model4(conv3_3[:,:,::2,::2])
        conv4_3 = self.weight_layer4(instance_feature['conv4_3'], conv4_3, box_info_list[3])
        ……`
